cmd/kubeadm/app/cmd/init.go

```diff
@@ -114,6 +110,7 @@ func NewCmdInit(out io.Writer) *cobra.Command {
 	var featureGatesString string
 	var criSocket string
 	var ignorePreflightErrors []string
+	var defaultUser string
```

Comment:
defaultUser doesn't say me anything about the purpose of this feature, which seems to be to copy `/etc/kubernetes/admin.conf` to `/home/$USER/.kube/config`.
Could we please come up with a better name for this?

Reply:
in the PR `additional info` i've added a comment that perhaps we should name this `--copy-config-for-user`. this also makes it kind of explicit so that the user knows that he/she is overwriting something in the home directory. if one does not want to overwrite then the flag should be simply omitted.